locales/en-US/app.ftl

```diff
@@ -846,6 +854,12 @@ TransformNavigator--focus-subtree = Focus Node: { $item }
 #   $item (String) - Name of the function that transform applied to.
 TransformNavigator--focus-function = Focus: { $item }
 
+# "Focus category" transform.
+# See: https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=focus-category
+# Variables:
+#   $item (String) - Name of the category that transform applied to.
+TransformNavigator--focus-category = Focus category: { $item }
```

Comment:
Please add in the comment that focus here is an adjective, as in "category currently focused", it's not a verb.

Reply:
Mmm is that so? We have a similar example above: for "Focus Node". I see that as a "reminder" of the action that was used. But I can see how this can be thought of as an adjective too... I guess both works...

Reply:
Verbs imply action, e.g. in a button. I assume this is a label stating that the category is already focused.

Reply:
No. It is something like "Java (non-project)" or "Layout".

Reply:
@parttimenerd, flod refer to "Focus" here.

Reply:
> Verbs imply action, e.g. in a button. I assume this is a label stating that the category is already focused.

now I kind of wonder if our english terms are correct :-) I guess we can think about that later...

Reply:
I added a comment `The word "Focus" has the meaning of an adjective here.`

Reply:
Hmm. "Focus category" is more like "Drop samples", it's cleary a verb in an assertive sentence.

Reply:
The item in the menu is a verb (action). Looking at the preview link, this is definitely a descriptive statement. If you were to write a sentence, it would be `The category currently focused is:`, not "Set the focus on this category:" (focus is already there).

![Schermata 2022-11-02 alle 10 42 44](https://user-images.githubusercontent.com/3644868/199457071-f1431a5d-cc6d-462f-8bff-894638b833ae.png)

Reply:
Ok

Reply:
> The item in the menu is a verb (action). Looking at the preview link, this is definitely a descriptive statement. If you were to write a sentence, it would be `The category currently focused is:`, not `Set the focus on this category:` (focus is already there).
> 
> ![Schermata 2022-11-02 alle 10 42 44](https://user-images.githubusercontent.com/3644868/199457071-f1431a5d-cc6d-462f-8bff-894638b833ae.png)

I see this as a breadcrumb trail, that is a reminder of the path we've taken to the current state. That's why I think that _in english_ I still prefer to keep the verb in this case, but for other languages it might be different